common/v2/features/PurchaseMembership/components/MembershipPurchaseForm.tsx

```diff
@@ -91,21 +96,6 @@ export const MembershipFormUI = ({ network, relevantAccounts, onComplete }: UIPr
       .min(0, translateRaw('ERROR_0'))
       .required(translateRaw('REQUIRED'))
       .typeError(translateRaw('ERROR_0'))
-      .test(
-        'check-amount',
-        translateRaw('BALANCE_TOO_LOW_NO_RECOMMENDATION_ERROR', { $asset: selectedAsset.ticker }),
-        function(value) {
-          const account = this.parent.account;
-          const asset = this.parent.asset;
-          const val = value ? value : 0;
-          if (!isEmpty(account)) {
-            return getAccountBalance(account, asset.type === 'base' ? undefined : asset).gte(
-              parseEther(val.toString())
-            );
-          }
-          return true;
-        }
-      )
```

Comment:
Is this not needed because we're pre-filtering the accounts?

Reply:
Exactly 😄